samples/python/create-invitations.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+
+###############################################################################
+# ex. python create-invitations.py --baseurl http://localhost:3000
+#       --username admin --password admin_pw
+#
+# To be run after submission due date to create review invitations for all the papers.
+# For each paper:
+# 1) create authorGroup (can see reviews, can't write a review)
+#           reviewer group (reviewers for this paper)
+#           and nonReviewerGroup (folks that aren't allowed to read the review at least not yet)
+# 2) create review invitation
+###############################################################################
+
+## Import statements
+import argparse
+import config
+from openreview import *
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+if args.username!=None and args.password!=None:
+    client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+else:
+    client = Client(baseurl=args.baseurl)
```

Comment:
in all these scripts, you can replace this if/else block with just a single line:

client = Client(baseurl=args.baseurl, username=args.username, password=args.password)

the Client object will search the environment variables for the username and password if they are None